.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,14 @@
+- id: check-xmllint
+  name: Running xmllint to check XML files format
+  description: Formats XML files
```

Comment:
This description should be different, such as "Check that XML files are correctly formatted".

I suggest not providing this option, unless it is common to do that. Do the black and isort hooks have a "look but don't touch" option? Having two ways to do things complicates the code, complicates the documentation (you need a section explaining how to manually format files) and makes the hook harder to use correctly. If the user pick the "read-only" option when they wanted the normal option then they (and all other editors of the package) will be annoyed.

Reply:
Black doesn't provide the option, as it cannot work that way. As we have the option, it's tempting to offer it. It might make sense for somebody outside of the project. I keep it, but marked it "don't use".